operations/agent-flow-mixin/dashboards/cluster-overview.libsonnet

```diff
@@ -99,7 +97,11 @@ local filename = 'agent-cluster-overview.json';
                     {
                       targetBlank: false,
                       title: 'Detail dashboard for node',
-                      url: '/d/agent-cluster-node/grafana-agent-flow-cluster-node?var-instance=${__data.fields.instance}&var-datasource=${datasource}&var-loki_datasource=${loki_datasource}&var-cluster=${cluster}&var-namespace=${namespace}',
+                      // Use this URL for development with Grizzly where links are not replaced with a UID.
+                      // url: '/d/agent-cluster-node/grafana-agent-flow-cluster-node?var-instance=${__data.fields.instance}&var-datasource=${datasource}&var-loki_datasource=${loki_datasource}&var-cluster=${cluster}&var-namespace=${namespace}',                      
+                      
+                      // Use this URL to render and deploy the mixin.
+                      url: '/d/%(uid)s/grafana-agent-flow-cluster-node?var-instance=${__data.fields.instance}&var-datasource=${datasource}&var-loki_datasource=${loki_datasource}&var-cluster=${cluster}&var-namespace=${namespace}' % {uid: std.md5(cluster_node_filename)},
```

Comment:
I absolutely hate this, but I'm not sure if there's a better way around it

The first link is what works when testing using Grizzly on a local Grafana instance. The second link works when the deployed dashboard gets the first part of the URL replaced with the linked dashboard's UID.

Any ideas here?

Reply:
It might work consistently if we change our Grizzly definition so that the name of [the Dashboard resource](https://github.com/grafana/agent/blob/9918ac74424d4a29b46b309f0cbc969e5b324be6/operations/agent-flow-mixin/grizzly.jsonnet#L31) is also the MD5?

Reply:
The UID is fixed to be the [MD5](https://github.com/grafana/agent/blob/9918ac74424d4a29b46b309f0cbc969e5b324be6/operations/agent-flow-mixin/dashboards/cluster-node.libsonnet#L13) so I think this will fix it.

Reply:
Yes, it works. It's a bit harder to understand which dashboard changed but the consistency between local development and rendered result makes it worth it IMHO.

```
INFO[0256] Changes detected. Applying ./grizzly.jsonnet
INFO[0256] Applying 6 resources
DashboardFolder.grafana-agent-flow updated
Dashboard.dd370cd333b2d9258435fb1b5a20a89b updated
Dashboard.7e07f9c975fcfc2a6e120a95f579f843 no differences
Dashboard.f861e5fef2e795edd5c4c73bee1ba769 no differences
Dashboard.513c25187e0289ec960f252466d5190d no differences
Dashboard.d47aae5c53be5550f8e3bc8a904ba61a no differences```